components/form/LabeledSelect.vue

```diff
@@ -200,7 +200,7 @@ export default {
       :options="options"
       :placeholder="placeholder"
       :reduce="x => reduce(x)"
-      :value="value ? value : ''"
+      :value="value ? value : false"
```

Comment:
This probably breaks some other usage that currently expects no value to be empty string (which is what standard html controls always are).

Reply:
I see, ok, updated so `undefined` or `null` are empty strings in `v-select`, but `false` will be  `false`(which was the issue in #1647 )